troubleshooting.md

```diff
@@ -45,4 +45,24 @@ tech side of things, you can read about Xero's API rate limits [here](https://de
 Subsequent syncs, either manual or automated overnight, only retrieve the Xero records that have changed since the last sync,
 so should be substantially faster in comparison.
 
-Think your import is taking too long or has failed? [Get in touch](/contact-us.html), we're here to help.
+Think your import is taking too long or has failed? See next section for synch failures or [Get in touch](/contact-us.html), we're here to help.
+
+## Initial Sync Failures
+
+Errors could occur when data is synchronized for the first time from Xero to Xavier.
+
+This section provides an overview of different types of sync errors, some of the possible scenarios that cause those errors and potential ways to fix the errors.
+
+This section includes the common error types and may not cover all the possible errors.
+
+* **Xero auth failed. Could not sync client**: Authentication error occurs when the provided credentials for the User seem to be invalid. Please try to refresh your Token or create/get a new Authentication key. Please see [Permissions](/troubleshooting.html#xero-access-permissions) above.
```

Comment:
This one might be a bit too technical for the user, I don't think they will know what the token or authentication key are. Maybe reword it to:

`Check that you are using the correct Xero account credentials, and have the required access rights to the client in Xero, before trying again. Please see [Permissions](/troubleshooting.html#xero-access-permissions) above for more information.`